types/TileInput.ts

```diff
@@ -24,6 +24,16 @@ export interface TileInput<T> extends TileRequest {
    */
   radius?: number;
 
+  /**
+   * @description The tile extent. Here the grid dimensions will be 4096x4096
```

Comment:
The tile extent is the grid dimension value as specified by `ST_AsMVT` https://postgis.net/docs/ST_AsMVT.html. The default is 4096.